Examples/ElizaCocoaPodsApp/Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
   - Connect-Swift (0.6.0):
-    - SwiftProtobuf (~> 1.22.0)
-  - SwiftProtobuf (1.22.0)
+    - SwiftProtobuf (~> 1.23.0)
+  - SwiftProtobuf (1.23.0)
```

Comment:
I assume that this indentation is autogenerated and correct...

Reply:
Yep